test-dts/tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why do we need this build config? (What are we testing with this?)

Reply:
This tests the output files, to make sure our output typings are correct. Because of the usage of `@internal` we might hide some `apis`, using this config will make the dependencies to resolve to the publish dirs instead of the src code